src/main/java/salesforce/ui/pages/quicktext/NewQuickText.java

```diff
@@ -11,34 +11,101 @@
  */
 public class NewQuickText extends BasePage {
 
+    private static final String COMBO_BOX = "//span[text() ='%s']/../..//a[@role='button']";
```

Comment:
instead of doing som /../.. I think you could use the //.. to go double parent.

Reply:
Its true